### PR TITLE
check for two approvals

### PR DESCRIPTION
**Pull request by lstod**
reopen

### CODE REVIEW

File: bin/README.md [hunk 1]
```diff
@@ -16,3 +16,4 @@ fi
 ## Commands
 
 * `alcs docker [up/down] (--metabase)`
+..
```
Comment (mhuseinov) [inline bin/README.md:19]: please remove this

**Review by mhuseinov — CHANGES_REQUESTED** [review]
please address comments

**Review by mhuseinov — DISMISSED** [review]
another request

**Comment by lstod** [comment]
😱

**Pull request closed without merging**